### DISCUSSION TITLE
[버그] 삭제화면 을/를 표기

### DISCUSSION THREAD

**Discussion by frin0911** [koreanbots/koreanbots#464] [제안]
![image](https://user-images.githubusercontent.com/44210830/128631606-22b8ba4a-1ef0-4743-a75c-a2d8b3d2cfcd.png)

봇 이름에 따라 을이 쓰일지 를이 쓰일지가 달라집니다.

**Comment by wonderlandpark** [comment]
한글을 기준으로 하여 을/를이 유연하게 표시됩니다. 따라서 버그가 아닌 제안사항으로 옮기도록 하겠습니다.
